Agreements/Internal/5_MAW.txt

```diff
@@ -17,6 +17,8 @@ COPX:*:*:BERUL:EGNT:*:London S5:Scottish PC West:*:28000:BERUL
 COPX:*:*:BERUL:EGNV:*:London S5:Scottish PC West:*:28000:BERUL
 COPX:*:*:BERUL:EGNJ:*:London S5:Scottish PC West:*:28000:BERUL
 
+COPX:*:*:NOKIN25:EGNT:*:London S5:Scottish PC West:*:28000:NOKIN25
```

Comment:
Probably covered in line 12, no problem - delete new line 20

Reply:
Allright.